src/main/java/fi/metatavu/soteapi/wordpress/tasks/AbstractPostUpdateModel.java

```diff
@@ -12,20 +12,24 @@
   private String originId;
   private String parentOriginId;
   private Long categoryId;
+  private String created;
+  private String modified;
   private Long orderIndex;
 
   public AbstractPostUpdateModel() {
   }
-
+  
   public AbstractPostUpdateModel(String title, String content, String slug, String originId, String parentOriginId,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)